go/enclave/db/rawdb/accessors_chain.go

```diff
@@ -179,16 +181,16 @@ func WriteBlockState(db ethdb.KeyValueWriter, bs *core.BlockState, logger gethlo
 	}
 }
 
-func ReadBlockState(kv ethdb.KeyValueReader, hash gethcommon.Hash, logger gethlog.Logger) *core.BlockState {
+func ReadBlockState(kv ethdb.KeyValueReader, hash gethcommon.Hash) (*core.BlockState, error) {
 	data, _ := kv.Get(blockStateKey(hash))
 	if data == nil {
-		return nil
```

Comment:
Can a TODO handle `kv.Get` error be added ?

Reply:
Just handled the error.